docs/docs/20-quickstart.md

```diff
@@ -568,269 +546,42 @@ the previous section.
     version of the `public.ecr.aws/nginx/nginx` container image.
     :::
 
-1. We'll use it later, so save the ID of the `Freight` to an environment
-   variable:
-
-    ```shell
-    export FREIGHT_ALIAS=$(kargo get freight --project kargo-demo --output jsonpath={.alias})
-    ```
-
-1. Now, let's _promote_ the `Freight` into the `test` `Stage`:
-
-    ```shell
-    kargo promote --project kargo-demo --freight-alias $FREIGHT_ALIAS --stage test
-    ```
-
-    Sample output:
-
-    ```shell
-    promotion.kargo.akuity.io/test.01j2wbtrym4r3tktv38qzteh5h.7a6e91f promotion created
-    ```
-
-    Query for `Promotion` resources within our project to see one has been
-    created:
+### Promote to Test
```

Comment:
To match my suggestion earlier of "Your First Project" we might consider renaming this to match:

```suggestion
### Your First Promotion
```